libraries/shared/watcher/storage_watcher.go

```diff
@@ -32,7 +32,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var ResultsLimit = 500
+var (
+	ResultsLimit       = 500
+	BlocksBackFromHead = int64(500)
```

Comment:
Thinking that we might want this number to be a good bit higher - at least the first time we run this, since it's possible we could be at a really bad point in terms of cycling through the unchecked (and untransformable diffs). I wonder if maybe instead of `skipOldDiffs` being a bool, it could be an int64 configurable via the CLI? Then we could easily bump the number up or down depending on the results we observe

Reply:
I really like the idea of passing in an instead!